pkg/cmd/server/origin/master.go

```diff
@@ -361,6 +361,7 @@ func (c *MasterConfig) InstallProtectedAPI(container *restful.Container) []strin
 	initControllerRoutes(root, "/controllers", c.Options.Controllers != configapi.ControllersDisabled, c.ControllerPlug)
 	initAPIVersionRoute(root, LegacyOpenShiftAPIPrefix, legacyAPIVersions...)
 	initAPIVersionRoute(root, OpenShiftAPIPrefix, currentAPIVersions...)
+	initReadinessCheckRoute(root, "/ready", c.ProjectAuthorizationCache.ReadyForAccess)
```

Comment:
minor change - can you make this "/healthz/ready"?  We had a discussion about it upstream and we agreed readiness is like healthz.  You may want to make the NonResourceURLs be properly wild carded (david would know how to do that) for healthz instead of explicitly enumerating the paths